### PR TITLE
Finishing the print_board function

### PR DESCRIPTION
**Pull request by miguelandresff**
@darieldejesus

### CODE REVIEW

File: tictactoe.py [hunk 1]
```diff
@@ -3,22 +3,32 @@
...
-                    print(" | ", end="")
+            print()
+            print('----------------------', end='')
+            print()
 
-        print()
-        print('----------------------',end='')
-        print()
 
+# Main program
```
Comment (darieldejesus) [inline tictactoe.py]: If this is the main call to execute the program, let's use __main__
https://docs.python.org/3/library/__main__.html#module-__main__

**Comment by miguelandresff** [comment]
@darieldejesus i made the changes, please check.

File: tictactoe.py [hunk 2]
```diff
@@ -1,24 +1,24 @@
 board = [[None, None, None], [None, None, None], [None, None, None]]
 
+
```
Comment (darieldejesus) [inline tictactoe.py]: This line shouldn't be there.

File: tictactoe.py [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
+                print('  {}  '.format(board[row][column]), end='')
+            if column <= 1:  # To draw verticals lines of the board
+                print(' | ', end='')
+        if row <= 1:  # To draw horizontal lines of the board
+            print()
+            print('----------------------', end='')
+            print()
+
+
+if __name__ == "__main__":
```
Comment (darieldejesus) [inline tictactoe.py]: 👍

**Comment by darieldejesus** [comment]
After this quick changes, this is ready to be merged!

**Pull request merged**